test/results_test.dart

```diff
@@ -138,7 +138,7 @@ Future<void> main([List<String>? args]) async {
     realm.close();
   });
 
-  test('Results iteration test', () {
+  test('Results iterating test', () {
```

Comment:
I think _iteration_ is the better word here, but then again english is not my native tongue.

Reply:
Done